### PR TITLE
Adds Esumsium to the game

### PR DESCRIPTION
**Pull request by Manly-Dwarf**
:cl: KawaiiBigBoss
add: Adds Esumsium, a basic mutation based off of our least favorite Discord dweller.
/:cl:

**_Don't merge this_**
fuck u esums
Bawb told me to put that last sentence in.
This is my first PR so this is more or less a test

### CODE REVIEW

File: hippiestation/code/datums/mutations.dm [hunk 1]
```diff
@@ -70,3 +70,36 @@
...
+	text_gain_indication = "<span class='danger'>You feel like you're smarter than everyone else!</span>"
+
+/datum/mutation/human/esumsium/on_life(mob/living/carbon/human/owner)
+	if(prob(10) && owner.stat == CONSCIOUS)
+		owner.Stun(200)
+		switch(rand(1, 3))
+			if(1)
+				owner.emote("drool")
+			if(2 to 3)
+				owner.say("[prob(50) ? ";" : ""][pick("MY IQ 130", "LITTLE GIRLS", "U IQ 85 LOL", "INTERNET SIGMUND FREUD", "BLOCKED LOL", "", "PORT LIFEWEB", "HIPPIESTAYSHUN IS DED", "BLATANTLY OBVIOUS METAGAMING", "I PROMISE I'M NOT INSECURE I'M JUST BETTER THAN YOU", " I A ROBOT BEEPBOOP x3")]")
```
Comment (ghost) [inline hippiestation/code/datums/mutations.dm:88]: " I A ROBOT BEEPBOOP x3" is that meant to be a smiley in game (which is against the rules) or beep boop times 3?

Comment (Manly-Dwarf) [reply hippiestation/code/datums/mutations.dm:88]: beep boop times 3
didn't even realize that it was a smiley but that only makes it funnier

File: hippiestation/code/datums/mutations.dm [hunk 2]
```diff
@@ -70,3 +70,36 @@
...
+		var/x_offset_old = owner.pixel_x
+		var/y_offset_old = owner.pixel_y
+		var/x_offset = owner.pixel_x + rand(-2,2)
+		var/y_offset = owner.pixel_y + rand(-1,1)
+		animate(owner, pixel_x = x_offset, pixel_y = y_offset, time = 1)
+		animate(owner, pixel_x = x_offset_old, pixel_y = y_offset_old, time = 1)
+
+	if((prob(15) && owner.IsUnconscious()))
```
Comment (ghost) [inline hippiestation/code/datums/mutations.dm:96]: why is there a check to see if the user is Unconcious before running code that only works when they are concious?

Comment (Manly-Dwarf) [reply hippiestation/code/datums/mutations.dm:96]: doesn't the tourettes seizure knock you unconscious?
it was basically there to give them braindamage whenever they had a seizure
edit: it doesn't

**Comment by Sarcalogo** [comment]
I think petty jabs at other players should be kept outside of the actual game code. We'll say I don't consent to the PR being merged. Imagine if someone did the same to you? Would you think it's okay for it to be added? Probably not.

That being said from a design perspective this makes no sense. 

- Use proper grammar and punctuation, the quotes are especially uncharacteristic otherwise.
- 130 should be changed to 130+, 130 makes no sense.
- "LITTLE GIRLS" should be removed, as its slander.
- "U IQ 85 LOL" again, no sense. Change to low IQ instead.
-"I PROMISE I'M NOT INSECURE I'M JUST BETTER THAN YOU" doesn't make sense, this is an obvious jab and not some sort of humorous remark. 
- "BLATANTLY OBVIOUS METAGAMING" - Should be changed to blatant and obvious metagaming all caps are fine.

Or maybe you could just wait and til I leave to add such foolishness?

**Comment by Iamgoofball** [comment]
>slander

https://file.house/Xu2y.png
https://file.house/Lqlw.png

**Comment by Iamgoofball** [comment]
ps @Sarcalogo if you're going to talk about leaving so much, Just Do It(tm)

**Comment by Manly-Dwarf** [comment]
@Sarcalogo 
Don't worry, this is a test PR.
Added "Don't merge" just in case

**Pull request closed without merging**